service/discovery/azure/compute.go

```
@@ -70,10 +70,31 @@ func (d *azureComputeDiscovery) List() (list []voc.IsResource, err error) {
 	}
 
 	// Discover virtual machines
+	virtualMachines, _ := d.discoverVirtualMachines()
```

Hm. This basically ignores the error in a higher level than before :)

Should be split this into individual discoverers?

---

Why? The error was already handled in the discoverVirtualMachines() function.